R/gs_power_ahr.R

```diff
@@ -171,7 +172,7 @@ gs_power_ahr <- function(
     info_scale = c("h0_h1_info", "h0_info", "h1_info"),
     r = 18,
     tol = 1e-6,
-    interval = c(.01, 100)) {
+    interval = c(.01, if(is.null(analysis_time)) 100 else max(analysis_time))) {
```

Comment:
Note that I used `max(analysis_time)` here simply because it is more readable than `analysis_time[length(analysis_time)]`